### ISSUE TITLE
Tutorial Feedback: Transform/Scale/Rotate should be tooltips not modals

### ISSUE THREAD

**Issue by netpro2k** [MozillaReality/Spoke#573]
Bit annoying that I have to go through all of them before getting to try any of them

**Comment by robertlong** [comment]
Yeah, @misslivirose maybe we can go through possible designs for this? I want to keep all the content, but as @netpro2k said I don't want to block people from trying things out immediately after learning them. I tried to mitigate this by sticking the hotkeys in a tooltip at the end, but it still isn't great.
